src/main/java/com/welovecoding/nbeditorconfig/listener/ProjectChangeListener.java

```diff
@@ -25,45 +25,47 @@ public ProjectChangeListener(Project project) {
   }
 
   @Override
+  public void fileFolderCreated(FileEvent event) {
+    super.fileFolderCreated(event);
+    LOG.log(Level.INFO, "PROJECTLISTENER: Created folder: {0}", event.getFile().getPath());
```

Comment:
Actually I did not changed or created the logger message itself but I can change it if you like :)